.github/workflows/main.yml

```diff
@@ -0,0 +1,42 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+    environment: 
+      name: staging
+      url: https://github.com
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      # Runs a single command using the runners shell
+      - name: build and push
+        env: # Or as an environment variable
+          AWS_ACCOUNT_ID: ${{ secrets.AWS_ACCOUNT_ID }}    
+          AWS_REGION: ${{ secrets.AWS_REGION }}      
+        run: |
+          cd $GITHUB_WORKSPACE
+          bash deployment/build_and_push.sh
+
+
+      # Runs a set of commands using the runners shell
+      - name: deploy
+        env: # Or as an environment variable
+          AWS_ACCOUNT_ID: ${{ secrets.AWS_ACCOUNT_ID }}    
+          AWS_REGION: ${{ secrets.AWS_REGION }} 
+          LAYER_NAME: ${{ secrets.LAYER_NAME }}
+          TARGET_GROUP_ID: ${{ secrets.TARGET_GROUP_ID }}
+        run: bash deployment/deploy.sh
```

Comment:
How did you ensure that this step has aws cli v2 and ecs-cli?

Reply:
from what i can tell the ubuntu runner has aws cli
added installation for ecs-cli